rpc/interface/src/router_builder.rs

```diff
@@ -0,0 +1,198 @@
+    other_get_transaction_pool        => "/get_transaction_pool"        => other::get_transaction_pool        => (get, post),
+    other_get_transaction_pool_hashes => "/get_transaction_pool_hashes" => other::get_transaction_pool_hashes => (get, post),
+    other_get_transaction_pool_stats  => "/get_transaction_pool_stats"  => other::get_transaction_pool_stats  => (get, post),
+    other_set_bootstrap_daemon        => "/set_bootstrap_daemon"        => other::set_bootstrap_daemon        => (get, post),
+    other_stop_daemon                 => "/stop_daemon"                 => other::stop_daemon                 => (get, post),
+    other_get_net_stats               => "/get_net_stats"               => other::get_net_stats               => (get, post),
+    other_get_limit                   => "/get_limit"                   => other::get_limit                   => (get, post),
+    other_set_limit                   => "/set_limit"                   => other::set_limit                   => (get, post),
+    other_out_peers                   => "/out_peers"                   => other::out_peers                   => (get, post),
+    other_in_peers                    => "/in_peers"                    => other::in_peers                    => (get, post),
+    other_get_outs                    => "/get_outs"                    => other::get_outs                    => (get, post),
+    other_update                      => "/update"                      => other::update                      => (get, post),
+    other_pop_blocks                  => "/pop_blocks"                  => other::pop_blocks                  => (get, post),
+
+    // Binary routes.
+    bin_get_blocks                  => "/get_blocks.bin"                  => bin::get_blocks                  => (get, post),
+    bin_getblocks                   => "/getblocks.bin"                   => bin::get_blocks                  => (get, post),
+    bin_get_blocks_by_height        => "/get_blocks_by_height.bin"        => bin::get_blocks_by_height        => (get, post),
+    bin_getblocks_by_height         => "/getblocks_by_height.bin"         => bin::get_blocks_by_height        => (get, post),
+    bin_get_hashes                  => "/get_hashes.bin"                  => bin::get_hashes                  => (get, post),
+    bin_gethashes                   => "/gethashes.bin"                   => bin::get_hashes                  => (get, post),
+    bin_get_o_indexes               => "/get_o_indexes.bin"               => bin::get_o_indexes               => (get, post),
+    bin_get_outs                    => "/get_outs.bin"                    => bin::get_outs                    => (get, post),
+    bin_get_transaction_pool_hashes => "/get_transaction_pool_hashes.bin" => bin::get_transaction_pool_hashes => (get, post),
+    bin_get_output_distribution     => "/get_output_distribution.bin"     => bin::get_output_distribution     => (get, post),
+}
```

Comment:
Route map.

builder function name => actual route literal => async route function => `$(supported HTTP methods)*`.

`monerod` seems to be pretty loose with what HTTP methods are supported, e.g. this works:
```bash
curl -v http://127.0.0.1:18081/get_limit --request DELETE -H 'Content-Type: application/json'
```
Some of these also don't make sense as a `get` since they require body content. Will fix alongside creating the handlers.